src/app/pages/rpc-send-transfer/components/send-transfer-confirmation-details.tsx

```diff
@@ -17,24 +17,24 @@ export function SendTransferConfirmationDetails({
   return (
     <Stack border="active" borderRadius="sm" p="space.05" gap="space.04" width="100%">
       <HStack alignItems="center" gap="space.04" justifyContent="space-between">
-        <styled.span color="accent.styled.span-subdued">From</styled.span>
+        <styled.span color="ink.styled.span-subdued">From</styled.span>
```

Comment:
This looks like an error from a precious refactor? Where someone (probably me) replaced `span` with `styled.span` 

So I think that we need to fix these colours to be:
```
color="ink.span-subdued"
```

Instead of 
```
color="ink.styled.span-subdued"
```

Reply:
Ok, nice catch, I don't even think we have a `span-subdued`. Haha. Should be `text-subdued`.

Reply:
Sounds like `sup, dude`